x-pack/platform/packages/shared/kbn-ai-assistant/src/hooks/use_conversation.ts

```diff
@@ -110,6 +117,29 @@ export function useConversation({
       });
   };
 
+  const fork = () => {
+    if (!displayedConversationId || !conversation.value) {
+      throw new Error('Cannot save fork if conversation is not stored');
+    }
+    return service
+      .callApi(`POST /internal/observability_ai_assistant/conversation/{conversationId}/fork`, {
+        signal: null,
+        params: {
+          path: {
+            conversationId: displayedConversationId,
+          },
+        },
+      })
+      .catch((err) => {
+        notifications!.toasts.addError(err, {
+          title: i18n.translate('xpack.aiAssistant.errorForkingConversation', {
+            defaultMessage: 'Could not fork conversation',
```

Comment:
I think we should get rid of the "fork" terminology everywhere
```suggestion
            defaultMessage: 'Could not duplicate conversation',
```

Reply:
done